apps/src/lib/script-editor/LevelTokenDetails.jsx

```diff
@@ -183,6 +186,13 @@ const LevelTokenDetails = React.createClass({
             />
           </div>
         )}
+        {/* We don't currently support editing progression names here, but do
+          * show the current progression if we have one. */}
+        {this.props.level.progression &&
+          <div style={styles.progression}>
+            Progression Name: {this.props.level.progression}
+          </div>
+        }
```

Comment:
Validated that I can go to http://localhost-studio.code.org:3000/s/csp1/edit?beta=true, save the script, and still have my progressions in place.